packages/admin/cms-admin/src/common/SearchInput.tsx

```diff
@@ -93,16 +93,16 @@ export const SearchInput = ({
             endAdornment={
                 internalQuery ? (
                     <InputAdornment position="end">
-                        <Typography>
+                        <Typography variant="body2" sx={{ marginRight: 2 }}>
                             {currentMatch !== undefined && totalMatches !== undefined ? `${currentMatch + 1}/${totalMatches}` : "..."}
                         </Typography>
-                        <IconButton onClick={jumpToPreviousMatch} disabled={!jumpToPreviousMatch} size="large">
-                            <ChevronUp />
+                        <IconButton onClick={jumpToPreviousMatch} disabled={!jumpToPreviousMatch} size="medium">
+                            <ChevronUp color="secondary" />
```

Comment:
Please always set the color on the `IconButton` instead of the icon so to hover/focus styling has the correct color as well. The icon will use that color automatically then. 

```suggestion
                        <IconButton onClick={jumpToPreviousMatch} disabled={!jumpToPreviousMatch} size="medium" color="secondary">
                            <ChevronUp />
```